pkg/kncloudevents/message_sender_test.go

```diff
@@ -180,6 +184,86 @@ func TestHTTPMessageSenderSendWithRetries(t *testing.T) {
 	}
 }
 
+func TestRetriesOnNetworkErrors(t *testing.T) {
+
+	port := rand.Int31n(math.MaxUint16-1024) + 1024
```

Comment:
I am not sure if this is a good strategy, this might result in some well known used port like 8080, in which case this test will fail randomly though rarely. IMO it is preferred to have a fixed port, at least when it fails it is easy to debug.

Reply:
Done! Thanks!